bin/fuzz.py

```diff
@@ -86,9 +86,8 @@ def _run(self, bar):
                     return False
                 localbar.done()
             else:
-                if not target_ansfile.is_file():
-                    bar.error(f'{self.i}: {ansfile.name} does not exist and was not generated.')
-                    return False
+                bar.error(f'{self.i}: {ansfile.name} does not exist and was not generated.')
```

Comment:
Update message `{ansfile.name} was not generated.`

Reply:
Done!